pages/common.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * NOTE: This file contains supplementary code shared by both "shows page" and "episodes page"
+ */
+
+// create some initial stage static page html content
+function createInitialContent(type) {
+  const rootElem = document.getElementById("root");
+  if (type === "shows") {
+    rootElem.innerHTML = initialShowsPageContent;
+  } else if (type === "episodes") {
+    rootElem.innerHTML = initialEpisodesContent;
+  } else {
+    rootElem.innerHTML = errorContent;
+  }
+}
+
+// set initial styling to some page components
+function setInitialStyling(pageName) {
+  const contentDiv = document.querySelector(`.all-${pageName}`);
+  const thumbnails = contentDiv.childNodes;
```

Comment:
why did you name your variable thumbnails. Just curious

Reply:
It's simply because the containers look a bit like thumbnails when you see them, especially the ones in the episodes page. :-)